xml/networking-isolated_metadata.xml

```diff
@@ -11,20 +11,23 @@
   In &productname;, Neutron currently sets <literal>enable_isolated_metadata =
```

Comment:
"Neutron currently sets" - this isn't exactly true, neutron defaults this to false. SUSE cloud sets it to true. https://docs.openstack.org/neutron/rocky/configuration/dhcp-agent.html

Reply:
@rktidwell , you missed the first part of the sentence that explains exactly that 

 ```In &productname;, Neutron currently sets ```

Reply:
I did misunderstood that. Thanks for setting me straight, this all makes perfect sense now.